dbt_coves/templates/source_model_props.yml

```diff
@@ -15,9 +15,19 @@ models:
     columns:
 {%- for cols in nested.values() %}
   {%- for col in cols %}
+  {%- if with_metadata == 'yes' %}
```

Comment:
`with_metadata` should not exist. Imagine that in the csv not all rows are specified. This will generate empty "description" keys.